error-messages/user.messages.js

```diff
@@ -7,6 +7,7 @@ const EMAIL_ERROR = 'EMAIL_ERROR';
 const INVALID_PERMISSIONS = 'INVALID_PERMISSIONS';
 const RECOVERY_ATTEMPTS_LIMIT_EXCEEDED = 'RECOVERY_ATTEMPTS_LIMIT_EXCEEDED';
 const TOKEN_NOT_VALID = 'TOKEN_NOT_VALID';
+const USER_ALREADY_CONFIRMED = 'USER_ALREADY_CONFIRMED';
```

Comment:
Please, rename it to `USER_EMAIL_ALREADY_CONFIRMED`

Reply:
renamed